accepted/0000-timezones.md

```diff
@@ -0,0 +1,102 @@
+- Feature Name: timezones
+- Start Date: 2019-09-16
+- RFC PR: 
+
+# Summary
+[summary]: #summary
+
+Timezones can be a big problem when managing an application like Uyuni that can be deployed and accessed from anywhere in the world. 
+If we don’t have some rules on how to display and handle dates, this can become quite a big mess to manage. 
+
+
+# Motivation
+[motivation]: #motivation
+
+The goal of this RFC is to define those rules and serve as a reference for the future.
+
+These rules should define a consistent way of display dates on the UI and make this information as clear as possible for the user.
+
+The user should be able to easily understand in which timezone a date is displayed and aware if there is any difference from the timezone
```

Comment:
humm we could remove the timezone information there, but I would not overcomplicate, that's not an easy thing to implement, because most of the time dates are being formatted server-side. IMO it's better to always show it to avoid misunderstandings.